### PR TITLE
Remove unused Contributor Card code

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: static/js/themeSwitcher.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
+/* eslint-disable no-var */
```
Comment (nschonni) [inline static/js/themeSwitcher.js]: Why?

Comment (XhmikosR) [reply static/js/themeSwitcher.js]: Because we only use `var`s. And this file wrongfully used const. Either we do it consistently everywhere and drop old IE support later, or we just stick to what we did.

**Review by Trott — APPROVED** [review]
I'd prefer we didn't disable the ESLint rule but won't block on that.

**Comment by Trott** [comment]
> I'd prefer we didn't disable the ESLint rule but won't block on that.

Oh, guess I should wait until it's ready-to-review before approving....

**Comment by XhmikosR** [comment]
> I'd prefer we didn't disable the ESLint rule but won't block on that.

Yeah, I think we should do that in a separate PR then and be consistent with it. Now it's a warning, it should be an error and applied everywhere.

> Oh, guess I should wait until it's ready-to-review before approving....

I need to test this a bit more, hopefully tomorrow.

File: static/js/main.js [hunk 2]
```diff
@@ -68,14 +68,15 @@
   });
 
   function setTheme(theme) {
-    document.querySelector('html').setAttribute(themeAttr, theme);
+    document.documentElement.setAttribute('data-theme', theme);
```
Comment (XhmikosR) [inline static/js/main.js]: This doesn't belong to this PR but I was too lazy to split it.
